example/Cpp/src/ROS/MinimalPublisher.lf

```diff
@@ -0,0 +1,31 @@
+target Cpp {
+    ros2: true
+}
+
+public preamble {=
+    #include "rclcpp/rclcpp.hpp"
+    #include "std_msgs/msg/string.hpp"
+=}
+
+main reactor {
+    private preamble {=
+        // FIXME: forward declaration to make the node visible
```

Comment:
Is this really necessary? Can't you generate this declaration?

Reply:
This is just a temporary workaround until I found a better solution. I am trying to keep the code for generating reactor classes agnostic of the target platform, as it keeps the code generator clean. However, there needs to be some way of extending the generated code. I will fix this mechanism as soon as I have found a satisfactory solution.